src/pages/AddRestaurantPage1/AddRestaurantPage1.jsx

```diff
@@ -0,0 +1,73 @@
+import CategoryButton from "../../components/shared/CategoryButton/CategoryButton";
+import Button from "../../components/shared/Button/Button";
+import "./AddRestaurantPage1.css";
+
+const titles = ["🌠 Wishlist", "💖 Favourites"];
```

Comment:
I would remove emoji for now, so we can set up backend and then maybe add them only on frontend